app/scripts/update-problem.js

```diff
@@ -0,0 +1,45 @@
+// module level exports
+const opn = require('opn');
+const {runApp} = require('../server/app');
+
```

Comment:
IMO, every import aside from `CONTENT` here is unnecessary.

Likewise, every import aside from `fsStat` is probably only import necessary in line 13